app/assets/javascripts/models/data/account.js.coffee

```diff
@@ -100,36 +38,24 @@ ns.Account = do (ko, ajax=@edsc.util.xhr.ajax) ->
           else
             @errors(["Contact information could not be loaded, please try again later"])
 
-    updateContactInformation: (callback) =>
+    updateNotificationPreference: (callback) =>
```

Comment:
This line and quite a few lines below are refactoring - users can only update the notification preferences now.